presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/testingServer/TestingArrowProducer.java

```diff
@@ -103,8 +101,7 @@ public void getStream(CallContext callContext, Ticket ticket, ServerStreamListen
             logger.debug("Executing query: %s", query);
 
             try (ResultSet resultSet = stmt.executeQuery(normalizeIdentifier(query))) {
-                JdbcToArrowConfig config = new JdbcToArrowConfigBuilder().setAllocator(allocator).setTargetBatchSize(2048)
-                        .setCalendar(Calendar.getInstance(TimeZone.getDefault())).build();
+                JdbcToArrowConfig config = new JdbcToArrowConfigBuilder().setAllocator(allocator).setTargetBatchSize(2048).build();
```

Comment:
Removed setting a calendar to retrieve Timestamps. It's not correct to use one in this case because values should not be converted to the Flight server local timezone. Existing tests were not affected because they were using the same time zone in the session anyway.